### DISCUSSION TITLE
i need to use karpor for my on prem database how i used the karpor for them. i m asking for just because it is already my sql app is running in my on prem

### DISCUSSION THREAD

**Discussion by vishu16392** [KusionStack/kusion#1240] [Q&A]
i need to use karpor for my on prem database how i used the karpor for them. i m asking for just because it is already my sql app is running in my on prem

**Comment by liu-hm19** [comment]
@elliotxx @ruquanzhao Please take a look at this question.

**Comment by elliotxx** [comment]
@vishu16392 Sorry, I just saw this question. Let me try to understand your problem, do you want to deploy Karpor in your on prem environment and connect to the existing database inside?
